[SSL] New FAQ page on CAs and certs (#10178)
cloudflare/cloudflare-docs: content/ssl/reference/certificate-authorities.md

{{<render file="_caa-records-definition.md">}}
<br/>

> If you are using Cloudflare as your DNS provider, then the CAA records will be added on your behalf. If you need to add CAA records, refer to the [Add CAA records page](/ssl/edge-certificates/caa-records/).

Reviewer: ```suggestion
If you are using Cloudflare as your DNS provider, then the CAA records will be added on your behalf. If you need to add CAA records, refer to [Add CAA records](/ssl/edge-certificates/caa-records/).
```
I usually don't add "page" to these references.

> You can see the CAA record content for each CA in the following table:

Reviewer: ```suggestion
The following table lists the CAA record content for each CA:
```
Avoiding "see".
